bugbug/model.py

```diff
@@ -254,8 +255,21 @@ def train(self, importance_cutoff=0.15):
 
         print(f"No confidence threshold - {len(y_test)} classified")
         confusion_matrix = metrics.confusion_matrix(y_test, y_pred, labels=class_names)
-        print(confusion_matrix)
         tracking_metrics["confusion_matrix"] = confusion_matrix.tolist()
+        confusion_matrix_reformatted = tracking_metrics["confusion_matrix"].copy()
```

Comment:
Call the variable `confusion_matrix_table`, and replace `tracking_metrics["confusion_matrix"].copy()` with `confusion_matrix.tolist()`